raiden/tests/utils/mediation_fees.py

```diff
@@ -109,4 +119,63 @@ def get_initial_payment_for_final_target_amount(
     except UndefinedMediationFee:
         return None
 
-    return PaymentWithFeeAmount(total), fees
+    return FeesCalculation(total_amount=PaymentWithFeeAmount(total), mediators_cut=fees)
+
+
+class PaymentAmountCalculation(NamedTuple):
+    """
+    Represents the result of get_amounts_to_drain_channel_with_fees
+
+    Differ from FeesCalculation in the fact that this returns the amount to send
+    without any fees or fee estimates.
+    """
+
+    amount_to_send: PaymentAmount
+    mediators_cut: List[FeeAmount]
```

Comment:
nitpick: should we call this `mediation_fees` for better clarity?

Reply:
Sure if you think it's a better name.